cli/args/mod.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+pub mod config_file;
+pub mod flags;
+
+mod flags_allow_net;
+
+pub use config_file::*;
+pub use flags::*;
```

Comment:
Can we get rid of wildcards here and explicitly list exports?

Reply:
Why not use wildcards?

Reply:
I'm planning to limit the config_file exports to reduce what's visible from this module in the next PR, but I think it's a lot easier to just do `pub use flags::*` for the flags.

Reply:
Okay sure